apps/src/gamelab/GameLab.js

```diff
@@ -308,6 +308,7 @@ GameLab.prototype.init = function (config) {
     isProjectLevel: !!config.level.isProjectLevel,
     isSubmittable: !!config.level.submittable,
     isSubmitted: !!config.level.submitted,
+    appType: 'gamelab'
```

Comment:
Rather than setting `appType` in each app, you should just set it once by passing in `appType: config.app` in the call to `setPageConstants` in `StudioApp.js`. Otherwise, looks good!